platform/openide.awt/src/org/openide/awt/SplittedPanel.java

```diff
@@ -94,13 +94,13 @@ public class SplittedPanel extends JComponent implements Accessible {
     public static final int HORIZONTAL = 2;
 
     /** constraints constant for adding a splitter */
-    public static final Object ADD_SPLITTER = new Integer(0);
+    public static final Object ADD_SPLITTER = new Object();
 
     /** constraints constant for adding a component to the first (left/top) pane */
-    public static final Object ADD_FIRST = new Integer(1);
+    public static final Object ADD_FIRST = new Object();
 
     /** constraints constant for adding a component to the second (right/bottom) pane */
-    public static final Object ADD_SECOND = new Integer(2);
+    public static final Object ADD_SECOND = new Object();
```

Comment:
this panel is public unfortunately, but unused and deprecated. This should be still compatible unless someone casts down to Integer.

Reply:
Could have kept 0, 1, and, 2...

Reply:
updated it